feat(mojaloop/#2743): documentation standards + contributors guide on node upgrade recommendations (#379)
mojaloop/documentation: docs/community/standards/creating-new-features.md

   npm run dep:update && npm i
   ```

2. REQUIRED - Vulnerability Checks

   ```bash
   npm run audit:check
   ```

>    If required, one can execute [npm audit](https://docs.npmjs.com/cli/v8/commands/npm-audit) to apply any known available fixes:

Reviewer: ```suggestion
[npm audit](https://docs.npmjs.com/cli/v8/commands/npm-audit) can be used to apply any known available fixes:
```
